node/test/graphql/query/me/training-data.spec.ts

```diff
@@ -26,21 +27,46 @@ describe('training data', () => {
     await mongoUnit.drop();
   });
 
-  it(`return all training data for lesson`, async () => {
+  it(`returns an error if not logged in`, async () => {
```

Comment:
we should test/confirm that only owner or contentManager + can view training data for a lesson